tests/framework/PostProcessors/InterfacedPostProcessor/test_interfacedPP_pointset.xml

```diff
@@ -79,10 +78,6 @@
     </MonteCarlo>
   </Samplers>
 
-  <Databases>
-    <HDF5 name="out_db" readMode="overwrite"/>
-  </Databases>
-
```

Comment:
same as above.

Reply:
I removed since it was not part of the testing capability